ext/mysql2/client.c

```diff
@@ -50,6 +52,12 @@ static ID intern_brackets, intern_merge, intern_merge_bang, intern_new_with_args
     rb_raise(cMysql2Error, "MySQL connection is already open"); \
   }
 
+#define REQUIRE_CONNECTED_ONCE(wrapper) \
+   REQUIRE_INITIALIZED(wrapper) \
+   if (!wrapper->client->server_version) { \
```

Comment:
Some others places in `client.c` use `wrapper->server_version`, is there a difference with `wrapper->client->server_version`?